src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -160,6 +166,18 @@ export class DictionaryConcepts extends Component {
     }
   }
 
+  handleDelete = () => {
+    const {
+      data, collectionName, type,
+    } = this.state;
+    this.props.removeDictionaryConcept(data, type, this.props.match.params.typeName, collectionName)
+      .then(setTimeout(() => {
+        this.props.fetchDictionaryConcepts(type, this.props.match.params.typeName, collectionName);
+      }, 2000));
```

Comment:
That is a very good observation. Instead of using .then to call the fetchDictionaryConcept prop I opted to use Redux reducers to handle updating of the state instead of making an extra call. Thus utilising the React hot-reloading feature.